store/datastore/secret.go

```diff
@@ -2,52 +2,32 @@ package datastore
 
 import (
 	"github.com/drone/drone/model"
-	"github.com/russross/meddler"
 )
 
-func (db *datastore) GetSecretList(repo *model.Repo) ([]*model.Secret, error) {
-	var secrets = []*model.Secret{}
-	var err = meddler.QueryAll(db, &secrets, rebind(secretListQuery), repo.ID)
-	return secrets, err
-}
+func (db *datastore) GetMergedSecretList(repo *model.Repo) ([]*model.Secret, error) {
```

Comment:
mentioned this below, but let's move this up a level. We can merge the two lists in `store` instead of `datastore`. That way, if we have multiple implementations of datastores, we don't need to duplicate the merge logic